pkg/ransim/model.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ransim
+
+import (
+	"context"
+	modelapi "github.com/onosproject/onos-api/go/onos/ransim/model"
+	"github.com/onosproject/onos-lib-go/pkg/cli"
+	"github.com/onosproject/onos-lib-go/pkg/errors"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"io/ioutil"
+)
+
+func loadCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "load",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Load model and/or metric data",
+		RunE:  runLoadCommand,
+	}
+	cmd.Flags().StringSlice("data-name", []string{}, "data set names")
+	cmd.Flags().StringSlice("data", []string{}, "data set file paths")
```

Comment:
How do we provide the files? Do we read them from local path? this is happening in onos-cli.

Reply:
Yes, local path.